source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_marl_env.py

```diff
@@ -563,6 +568,39 @@ def set_debug_vis(self, debug_vis: bool) -> bool:
     Helper functions.
     """
 
+    def _apply_scene_randomization(self):
```

Comment:
Can we make this a static helper method so that all 3 implementations can share it? It would need to take in an `env` of any of the three types and do the same operation on each